CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Next Release
 ### Feature
 - User can soft delete patient
+- Rename the button label to 'Next' when assigning a new BP passport to an existing patient
```

Comment:
One thing I am still unsure of is naming this section 'Feature' since this is definitely *not* a feature, but it also is *neither* an internal change nor a bug fix.

Maybe this section should be named "User Facing". Something to think about.

Reply:
Yeah, I was unsure about it as well. Although I am not entirely sure about "User Facing", perhaps we can name it as "Changes" and we can copy it to Play Store changelog as well. WDYT?

Reply:
Yeah, let's do this from the next release onwards.